amgut/lib/data_access/test/test_ag_data_access.py

```diff
@@ -610,70 +668,92 @@ def test_checkPrintResults_invalid_ids(self):
+        obs = self.ag_data.get_user_for_kit(
+            self.ag_data.get_supplied_kit_id(
+                'd8592c74-8421-2135-e040-8a80115d6401'))
         self.assertEqual('d8592c74-8421-2135-e040-8a80115d6401', obs)
 
     def test_get_user_for_kit_errors(self):
         with self.assertRaises(ValueError):
             self.ag_data.get_user_for_kit('the_fooster')
 
         with self.assertRaises(ValueError):
-            self.ag_data.get_user_for_kit('tst_esXXX')
+            self.ag_data.get_user_for_kit('NOT_IN_DB')
 
     def test_get_menu_items(self):
-        obs = self.ag_data.get_menu_items('tst_pDWcB')
+        obs = self.ag_data.get_menu_items(
+            self.ag_data.get_supplied_kit_id(
+                'd8592c74-844b-2135-e040-8a80115d6401'))
         self.assertEqual(({}, {}, [], True), obs)
 
-        obs = self.ag_data.get_menu_items('tst_VpQsT')
-        self.assertEqual(({'REMOVED-0': []}, {}, [], True), obs)
+        ag_login_id = 'd8592c74-84c9-2135-e040-8a80115d6401'
+        obs = self.ag_data.get_menu_items(
+            self.ag_data.get_supplied_kit_id(ag_login_id))
+        names = self.ag_data.get_participant_names_from_ag_login_id(
+            ag_login_id)
+        self.assertEqual((dict((name, []) for name in names),
+                         {}, [], True), obs)
 
     def test_get_menu_items_errors(self):
         with self.assertRaises(ValueError):
-            self.ag_data.get_menu_items('tst_esXXX')
+            self.ag_data.get_menu_items('NOT_IN_DB')
 
     def test_check_if_consent_exists(self):
+        ag_login_id = '00711b0a-67d6-0fed-e050-8a800c5d7570'
+        names = self.ag_data.get_participant_names_from_ag_login_id(
+            ag_login_id)
         obs = self.ag_data.check_if_consent_exists(
-            '00711b0a-67d6-0fed-e050-8a800c5d7570', 'REMOVED-42')
+            ag_login_id, names[0])
         self.assertTrue(obs)
 
     def test_check_if_consent_exists_non_existent_user(self):
-        obs = self.ag_data.check_if_consent_exists(
-            '00711b0a-67d6-0fed-e050-8a800c5d7570', 'REMOVED-111')
+        ag_login_id = '00711b0a-67d6-0fed-e050-8a800c5d7570'
+        obs = self.ag_data.check_if_consent_exists(ag_login_id, 'REMOVED-111')
         self.assertFalse(obs)
 
     def test_get_user_info(self):
-        obs = self.ag_data.get_user_info('tst_wAhSB')
-        exp = {'address': 'REMOVED', 'ag_login_id':
-               'd8592c74-84a5-2135-e040-8a80115d6401', 'city': 'REMOVED',
-               'country': 'REMOVED', 'email': 'REMOVED', 'name': 'REMOVED',
-               'state': 'REMOVED', 'zip': 'REMOVED'}
-        self.assertEqual(exp, obs)
+        ag_login_id = 'd8592c74-84a5-2135-e040-8a80115d6401'
+        obs = self.ag_data.get_user_info(
+            self.ag_data.get_supplied_kit_id(ag_login_id))
+        # unfortunatly, most fields are scrubbed in the database, thus we
+        # cannot compare them over DB versions
+        exp = {'ag_login_id': ag_login_id,
```

Comment:
Can we check in this case that at least all the keys that we are expecting are there?
It will be something like:
```python
exp = ['address', 'ag_login_id', 'city', 'country', 'email', 'name', 'state', 'zip']
self.assertItemsEqual(obs, exp)
```
Although `obs` is a dictionary, when used in `assertItemsEqual` it only looks at the keys.